app/test_split_integer.py

```diff
@@ -2,20 +2,24 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(20, 3)) == 20
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    result = split_integer(21, 3)
+    assert (max(result) - min(result)) == 0
```

Comment:
The assertion `(max(result) - min(result)) == 0` is incorrect for checking if the parts are equal. This assertion will only pass if all parts are exactly equal, which might not be the case if the value is not perfectly divisible by the number of parts. Consider checking if the parts are approximately equal or if the difference between the largest and smallest part is at most 1.